Source/WebKit/UIProcess/PageLoadState.h

```diff
@@ -216,6 +218,7 @@ class PageLoadState {
 
         String provisionalURL;
         String url;
+        WebCore::SecurityOriginData origin;
```

Comment:
Do we really need a new data member? Can’t we deduce the origin from the committed url instead?

Reply:
Not for opaque origins